set.mm

```diff
@@ -196532,6 +196669,34 @@ reduced fraction representation (no common factors, denominator
       TVD $.
   $}
 
+  $( ~ numdensq extended to nonnegative exponents.  (Contributed by Steven
```

Comment:
We should explain the theorem here, not only reference to another theorem. Maybe the same comment as for ~ numdensq.
The same for the following theorems.

Reply:
@tirix would you mind to extend the comments so that they are more explanatory? Comments in the mathboxes are often somehow sloppy, and they should be adjusted when they are moved to main to meet the higher standards for the main body.

Reply:
Done with https://github.com/metamath/set.mm/pull/4894/commits/cb716379b00c8344c97cb138777dcd78a0aa5e2b !